projects/coordinator/src/main/java/org/batfish/coordinator/WorkMgrServiceV2.java

```diff
@@ -29,40 +27,39 @@
 @Path(CoordConsts.SVC_CFG_WORK_MGR2)
 @Produces(MediaType.APPLICATION_JSON)
 public class WorkMgrServiceV2 {
+
+  private BatfishLogger _logger = Main.getLogger();
+
+  @DefaultValue(CoordConsts.DEFAULT_API_KEY)
+  @HeaderParam(CoordConsts.SVC_KEY_API_KEY)
+  private String _apiKey;
```

Comment:
Here I assume the key and version already been validated by filters, store apiKey for container accessibility check.